internal/controllers/machine/drain/drain.go

```diff
@@ -448,46 +448,50 @@ func (r EvictionResult) ConditionMessage(nodeDrainStartTime *metav1.Time) string
 
 	conditionMessage := fmt.Sprintf("Drain not completed yet (started at %s):", nodeDrainStartTime.Format(time.RFC3339))
 	if len(r.PodsDeletionTimestampSet) > 0 {
-		conditionMessage = fmt.Sprintf("%s\n* Pods with deletionTimestamp that still exist: %s",
-			conditionMessage, PodListToString(r.PodsDeletionTimestampSet, 3))
+		kind := "Pod"
+		if len(r.PodsToTriggerEvictionLater) > 1 {
+			kind = "Pods"
+		}
+		conditionMessage = fmt.Sprintf("%s\n* %s %s: deletionTimestamp set, but still not removed from the Node",
+			conditionMessage, kind, PodListToString(r.PodsDeletionTimestampSet, 3))
 	}
 	if len(r.PodsFailedEviction) > 0 {
 		sortedFailureMessages := maps.Keys(r.PodsFailedEviction)
 		sort.Strings(sortedFailureMessages)
 
-		conditionMessage = fmt.Sprintf("%s\n* Pods with eviction failed:", conditionMessage)
-
 		skippedFailureMessages := []string{}
 		if len(sortedFailureMessages) > 5 {
 			skippedFailureMessages = sortedFailureMessages[5:]
 			sortedFailureMessages = sortedFailureMessages[:5]
 		}
 		for _, failureMessage := range sortedFailureMessages {
 			pods := r.PodsFailedEviction[failureMessage]
-			conditionMessage = fmt.Sprintf("%s\n  * %s: %s", conditionMessage, failureMessage, PodListToString(pods, 3))
+			kind := "Pod"
+			if len(r.PodsFailedEviction[failureMessage]) > 1 {
```

Comment:
```suggestion
			if len(pods) > 1 {
```
nit: Doesn't change the result, but slightly better